format/spec.md

```diff
@@ -1267,6 +1304,7 @@ This serialization scheme is for storing single values as individual binary valu
 | **`fixed(L)`**               | Binary value                                                                                                 |
 | **`binary`**                 | Binary value (without length)                                                                                |
 | **`decimal(P, S)`**          | Stores unscaled value as two’s-complement big-endian binary, using the minimum number of bytes for the value |
+| **`variant`**                | Variant metadata bytes concatenated with variant value bytes                                                 |
```

Comment:
I think for non-shredded lower and upper bounds we should define this as not-supported. I think as we had some discussion before, this can be Variant metadata bytes + values. We'll probably have to define elsewhere how to actually calculate upper and lower bounds etc ....